examples/src/main/java/nl/knaw/dans/lib/dataverse/example/DataverseCreate.java

```diff
@@ -29,21 +30,28 @@
 public class DataverseCreate extends ExampleBase {
 
     public static void main(String[] args) throws Exception {
+        var dataverse = getDataverse();
+        log.info(toPrettyJson(dataverse));
+        DataverseHttpResponse<Dataverse> r = client.dataverse("root").create(dataverse);
+        log.info("Status Line: {} {}", r.getHttpResponse().getCode(), r.getHttpResponse().getReasonPhrase());
+        if (r.getHttpResponse().getCode() < HttpStatus.SC_CLIENT_ERROR) {
+            log.info("Created: {}", r.getData().getDescription());
+            log.info("Creation Date: {}", r.getData().getCreationDate());
+        }
+    }
+
+    public static Dataverse getDataverse() {
         Dataverse dataverse = new Dataverse();
         dataverse.setDataverseType(DataverseType.JOURNALS);
         dataverse.setName("A Test Dataverse");
         dataverse.setAlias("test2");
+        dataverse.setTheme(new DataverseTheme());
```

Comment:
Maybe we should not put fields with null values in a json string. See https://github.com/IQSS/dataverse/issues/11986
https://github.com/DANS-KNAW/dans-dataverse-client-lib/blob/a4829681fe96c6687527fac5f4719bcc7f07c4db/lib/src/main/java/nl/knaw/dans/lib/dataverse/HttpClientWrapper.java#L83-L85